packages/serverpod/lib/src/server/message_central.dart

```diff
@@ -7,10 +7,10 @@ abstract class MessageCentralServerpodChannels {
   /// Used to revoke authentication tokens.
   /// The message should be of type [RevokedAuthenticationUser],
   /// [RevokedAuthenticationAuthId] or [RevokedAuthenticationScope].
-  /// The [userId] should be the [AuthenticationInfo.userId] for the concerned
+  /// The user ID should be the [AuthenticationInfo.user] for the concerned
   /// user.
-  static String revokedAuthentication(int userId) =>
-      '_serverpod_revoked_authentication_$userId';
+  static String revokedAuthentication(Object user) =>
```

Comment:
I wonder if we should break this to `String user`, as this probably has very few callers outside of our packages.